### PR TITLE
updating the prototyping tool links

### PR DESCRIPTION
**Pull request by PtitBen**
- Added the playground as a first option
- Reframed the prototyping repository as an "advanced" option

### CODE REVIEW

File: docs/getting-started/working-with-nucleus.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ Depending on which stage of your design or discovery work you’re at, the Nucle
 
 * Familiarise yourself with this documentation and [Storybook](https://library.britishgas.design/).
 * Download the [Sketch library](https://github.com/ConnectedHomes/centrica-ux/releases).
-* Take a look at our [prototyping tool](https://github.com/britishgas-engineering/nucleus-prototype).
+* Take a look at our [prototyping tool](https://playground.nucleus.design).
```
Comment (andij) [inline docs/getting-started/working-with-nucleus.md:18]: Let's call this the Nucleus Playground
```suggestion
* Take a look at the [Nucleus Playground](https://playground.nucleus.design).
```

File: docs/getting-started/working-with-nucleus.md [hunk 2]
```diff
@@ -15,7 +15,8 @@ Depending on which stage of your design or discovery work you’re at, the Nucle
...
+* Or if you are familiar with GitHub, you can set yourself an environment on your local machine, [check out the prototyping repository](https://github.com/britishgas-engineering/nucleus-prototype)!
```
Comment (andij) [inline docs/getting-started/working-with-nucleus.md:19]:
```suggestion
* Or if you are familiar with GitHub, you can install the Prototyping Tool on your local machine, check out the [Nucleus Prototyping Tool repository](https://github.com/britishgas-engineering/nucleus-prototype)!
```

**Comment by PtitBen** [comment]
Nice suggestions Drew, thanks! 
All committed :)

**Review by andij — APPROVED** [review]

**Comment by andij** [comment]
Approved 👍

**Pull request merged**